.travis.yml

```diff
@@ -8,17 +8,23 @@ before_install:
   - echo "deb https://repos.mesosphere.io/ubuntu/ trusty main" | sudo tee /etc/apt/sources.list.d/mesosphere.list
   - sudo apt-get update -qq
   - sudo apt-get install mesos -y </dev/null
-  - pyenv install 3.6.2
+  - eval "$(pyenv init -)"
+  - pyenv install --skip-existing 3.6.2
```

Comment:
Is it possible to add comments in this file? If so, it might be nice to explain why we're doing some of this. If not, no big deal